chart/kubeapps/README.md

```diff
@@ -48,6 +47,8 @@ The command deploys Kubeapps on the Kubernetes cluster in the `kubeapps` namespa
 
 > **Tip**: List all releases using `helm list`
 
+Once you have installed Kubeapps follow the [Getting Started Guide](../docs/user/getting-started.md) for additional information on how to access and use Kubeapps.
```

Comment:
oh, this should be an absolute URL as this README can be rendered elsewhere (e.g. https://hub.kubeapps.com/charts/bitnami/kubeapps)